src/Cargo.toml

```diff
@@ -52,11 +52,15 @@ debug-assertions = false
 [patch."https://github.com/rust-lang/cargo"]
 cargo = { path = "tools/cargo" }
 
-[patch."https://github.com/rust-lang-nursery/rustfmt"]
+[patch.crates-io]
```

Comment:
why is this turned back to crates.io? Don't we want to override with the local rustfmt?

Reply:
Oh that's because the RLS [currently depends on `rustfmt-nightly` through crates.io](https://github.com/rust-lang-nursery/rls/blob/920a3914956caaa9e9facccdaca7114875247807/Cargo.toml#L35)